learn/configuration/typo_tolerance.md

```diff
@@ -0,0 +1,64 @@
+# Typo tolerance
+
+Typo tolerance allows users to find documents they're looking for even when they make mistakes while typing. Meilisearch allows you to [configure the typo tolerance feature for each index](/reference/api/typo_tolerance.md#update-typo-tolerance).
+
+:::note
+Meilisearch considers a typo on a query's first character as two typos. This increases performance during search.
+:::
+
+:::note
+Concatenating two strings will count as one typo.
+:::
+
+## Configuring typo tolerance
+
+Typo tolerance is enabled by default, but you can disable it if needed:
+
+<CodeSamples id="typo_tolerance_guide_1" />
+
+Disabling typo tolerance will mean Meilisearch no longer applies typo tolerance to match queries at search time. However, it will still [sort results by increasing number of typos](#impact-of-typo-tolerance-on-the-typo-ranking-rule).
+
+### `minWordSizeForTypos`
+
+By default, Meilisearch applies typo tolerance to a query if its length is at least 5 characters. To accept two typos, the query should be at least 9 characters.
+
+If your dataset contains `seven`, searching for `sevem` or `sevan` will match `seven`. But `tow` won't match `two` as it's less than 5 characters.
+
+You can override these default settings using the `minWordSizeForTypos` object. The code sample below sets the minimum word size for one typo to 4 and the minimum word size for two typos to 10.
+
+<CodeSamples id="typo_tolerance_guide_4" />
+
+When updating the `minWordSizeForTypos` object, keep in mind that:
+
+- the value of `twoTypos` should be greater or equal to `oneTypo`
+- the value for both `oneTypo` and `twoTypos` should be between `0` and `255`
+
+### `disableOnWords`
+
+You can disable typo tolerance for a list of query terms by adding them to `disableOnWords`.
+
+<CodeSamples id="typo_tolerance_guide_3" />
+
+Meilisearch won't apply typo tolerance on the query term `Shrek` or `shrek` at search time to match documents.
+
+:::note
+`disableOnWords` is case insensitive.
+:::
+
+### `disableOnAttributes`
+
+You can disable the typo tolerance feature for a specific attribute by adding it to `disableOnAttributes`. The code sample below disables typo tolerance for `title`:
+
+<CodeSamples id="typo_tolerance_guide_2" />
+
+If you make any typos in `title` at search, Meilisearch won't match any documents.
```

Comment:
It is a bit convoluted, feel free to simplify somehow, but this is actually how it works.
```suggestion
If you make a typo on a search term ("hallo" instead of "hello"), and "hello" is present in the `title` field of a document, then this occurrence of "hello" will not be a match at search time.
```